source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -28,13 +30,13 @@ OpenGLSurface: abstract class extends GpuSurface {
 	_unbind: abstract func
 	draw: override func (action: Func) {
 		this _bind()
-		this context _backend setViewport(this viewport)
+		this context backend setViewport(this viewport)
 		if (this opacity < 1.0f)
-			this context _backend blend(this opacity)
+			this context backend blend(this opacity)
 		else if (this blend)
-			this context _backend blend()
+			this context backend blend()
 		else
-			this context _backend enableBlend(false)
+			this context backend enableBlend(false)
```

Comment:
Yes, this is more of a quick fix than a long term solution. The problem here is the old `enableBlend` function which should be removed, but to prevent breaking the rendering of logo and version I have left it there for now. It will be taken care of in a future PR.